### PR TITLE
#Fhg-2732 what support does the service offer

### PR DESCRIPTION
**Pull request by witpo**

### CODE REVIEW

File: src/FamilyHubs.ServiceDirectory.Admin.Web/Pages/manage-services/Support-Offered.cshtml [hunk 1]
```diff
@@ -1,24 +1,70 @@
...
 }
 
+@{
+    Model.Errors.ErrorIdToHtmlElementId = _ => Model.UserInput.ErrorElementId;
+}
+
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-        <form method="post" novalidate>
+        <partial name="_ErrorSummaryNext" model="Model.Errors" />
```
Comment (lazcool) [inline src/FamilyHubs.ServiceDirectory.Admin.Web/Pages/manage-services/Support-Offered.cshtml:17]: This [partial](https://github.com/DFE-Digital/fh-web-components/blob/main/src/FamilyHubs.SharedKernel.Razor/Pages/Shared/_ErrorSummaryNext.cshtml) renders its own row and grid, so should be placed above the declared row.

Comment (witpo) [reply src/FamilyHubs.ServiceDirectory.Admin.Web/Pages/manage-services/Support-Offered.cshtml]: fixed

File: src/FamilyHubs.ServiceDirectory.Admin.Web/Pages/manage-services/Support-Offered.cshtml [hunk 2]
```diff
@@ -1,24 +1,70 @@
...
+
+        <form method="post" novalidate data-validate="my-form-validation">
             <h1 class="govuk-heading-l">
                 What support does the service offer?
             </h1>
+            <p>Select the categories and subcategories of support that provides. </p>
```
[the remark this answers is not in the document]
Comment (witpo) [reply src/FamilyHubs.ServiceDirectory.Admin.Web/Pages/manage-services/Support-Offered.cshtml]: Good spot, service name added

File: src/FamilyHubs.ServiceDirectory.Admin.Web/Pages/manage-services/Support-Offered.cshtml [hunk 3]
```diff
@@ -1,24 +1,70 @@
...
+@{
+    Model.Errors.ErrorIdToHtmlElementId = _ => Model.UserInput.ErrorElementId;
+}
+
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-        <form method="post" novalidate>
+        <partial name="_ErrorSummaryNext" model="Model.Errors" />
+
+        <form method="post" novalidate data-validate="my-form-validation">
```
[the remark this answers is not in the document]
Comment (witpo) [reply src/FamilyHubs.ServiceDirectory.Admin.Web/Pages/manage-services/Support-Offered.cshtml]: removed

**Review by lazcool — APPROVED** [review]

**Pull request merged**